jenkins/Jenkinsfile

```
@@ -93,33 +110,37 @@ pipeline {
               '''
 
               echo 'Save Artifacts'
-              sh("mkdir -p ${workspace}/rpm && cp -rf /root/rpmbuild/RPMS/x86_64/. ${workspace}/rpm")
-              dir("${workspace}/rpm") {
+              sh("mkdir -p ${WORKSPACE}/rpm && cp -rf /root/rpmbuild/RPMS/x86_64/. ${WORKSPACE}/rpm")
+              dir("${WORKSPACE}/rpm") {
                 archiveArtifacts artifacts: '*.rpm', onlyIfSuccessful: true
               }
             }
           }
         }
       }
     }
-    stage('Test'){
-      steps {
-        container('node'){
-          sh "mkdir -p ${workspace}/test && cp -rf ${workspace}/${stage_dir}/. ${workspace}/test"
-          dir("test") {
-            echo 'Run static code analysis and code style check'
-            sh 'npm run lint'
-
-            echo 'Check TypeScript types'
-            sh 'npm run check-types'
-
-            echo 'Run frontend tests'
-            sh 'npm run test:frontend'
-
-            echo 'Run backend tests'
-            sh 'sudo rm -rf /var/lib/apt/lists/* && sudo apt update && sudo apt install postgresql-client'
-            sh 'echo \'{"app":{"ssl":{"ca":null},"logLevelConf":"","logLevel":"debug"}}\' > conf/me.json'
-            sh 'npm run test:backend'
+    stage('Run Builds & Build RPM') {
```

This should probably say

```suggestion
    stage('Run tests') {
```

as the build step happens way earlier :slightly_smiling_face: